docs/topics/mta-web-administrator-view.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="mta-web-administration-view_{context}"]
+= User Interface views
+
+The {ProductName} ({ProductShortName}) {WebName} and has two user interfaces with different functionalities:
+
+* Administrator view
+* Developer view
+
+== Administrator view
+The administrator view is used by administrators to create the {ProductShortName} {WebName} instance environment.
+//![](/Tackle2/Views/AdminMenu.png)
+
+=== Credentials
+The credentials management module enables administrators to create and manage credentials to access private repositories.  It enables architects to assign credentials to applications without having to know their contents.
+//![](/Tackle2/Views/CredMain.png)
+
+The credentials page displays the available credentials, *Edit* and *Delete* buttons and the following fields:
+
+* *Name*
+* *Description*
+* *Type*
+** *Source Control*
+** *Username/Password*
+** *Source Private Key/Passphrase*
+** *Maven Settings File*
+** *Proxy*
+* *Type specific information*
+* *Created by*
+
+NOTE: Credential information entered, such as keys and passphrases, is hidden or displayed as [Encrypted].
+
+=== Repositories
+The repository management module configures the repositories used by {ProductShortName} {WebName} and has the following options:
+
+//![](/Tackle2/Views/GitConfig.png)
+* *Git configuration*
+** *Consume insecure Git repositories*
+//![](/Tackle2/Views/SubvConfig.png)
+* *Subversion configuration*
+** *Consume insecure Subversion repositories*
+//![](/Tackle2/Views/MavenConfig.png)
+
+* *Maven configuration*
+** Manage repository size
+** *Force update of dependencies*
```

Comment:
I don't think this is an option anymore. @jortel please confirm.

Reply:
@rromannissen waiting for a response.